source/common/config/filter_json.cc

```diff
@@ -289,6 +289,30 @@ void FilterJson::translateHealthCheckFilter(
   JSON_UTIL_SET_STRING(json_config, proto_config, endpoint);
 }
 
+void FilterJson::translateGrpcJsonTranscoder(
+    const Json::Object& json_config,
+    envoy::api::v2::filter::http::GrpcJsonTranscoder& proto_config) {
+  json_config.validateSchema(Json::Schema::GRPC_JSON_TRANSCODER_FILTER_SCHEMA);
+  JSON_UTIL_SET_STRING(json_config, proto_config, proto_descriptor);
+  auto* services = proto_config.mutable_services();
+  for (const auto& service_name : json_config.getStringArray("services")) {
+    *services->Add() = service_name;
+  }
+
+  if (json_config.hasObject("print_options")) {
+    auto json_print_options = json_config.getObject("print_options");
+    auto* proto_print_options = proto_config.mutable_print_options();
+    proto_print_options->set_add_whitespace(
```

Comment:
JSON_UTIL_SET_BOOL for all of these?

Reply:
I thought it only worked on wrapped types... let me try

Reply:
Nope.. it won't work.. the macro expects wrapped types..

Reply:
OK no problem. Thanks for trying.